### ISSUE TITLE
[Feature][WEBJS] - show reactions in GET /messages  (or in different API?)

### ISSUE THREAD

**Issue by HuangDaHui** [allburov/whatsapp-http-api#665]
/api/session_d3lfcm/chats/xxx%40g.us/messages?downloadMedia=false&limit=100
It seems that reactions cannot be obtained through the interface?
NOWEB It can be obtained, but WEBJS does not seem to see this field

**Comment by HuangDaHui** [comment]
Version:
2024.9.3

**Comment by devlikepro** [comment]
yes, it doesn't have it probably, reactions are different creatures in WEBJS. Will check that

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)
